rules/properties-order/tests/empty-line-before-unspecified.js

```diff
@@ -61,11 +61,57 @@ testRule(rule, {
 				}
 			`,
 		},
+		{
+			description: '6',
+			code: `
+				a {
+					height: 20px;
+					/* other props */
```

Comment:
Are you suggesting this should pass with `emptyLineBeforeUnspecified: 'always',` set?  I was under the impression this should fail.  Anything that isn't a 'height' or 'width' css property should classify as "unspecified" and therefore should have an empty line before it w/ the above config.

Reply:
We ignore requirement if there is a comment before. Take a look at these tests for `emptyLineBefore`:

https://github.com/hudochenkov/stylelint-order/blob/b0032196ade816005077234ea7cf8895300eca0b/rules/properties-order/tests/empty-line-before.js#L90-L99

https://github.com/hudochenkov/stylelint-order/blob/b0032196ade816005077234ea7cf8895300eca0b/rules/properties-order/tests/empty-line-before.js#L111-L122

They are both accepted when `emptyLineBefore: "always"` is set for `position`.

Reply:
Word “unspecified” in `properties-order` rule always mean properties which are not specified in a config.

Reply:
Thanks for clarifying, I should have a PR up soon for this then.